CHANGELOG.md

```diff
@@ -5,14 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
```

Comment:
We normally keep the Unreleased heading at the top so we can add to it as we go. Could you add this back please?

Reply:
Done